karma.conf.js

```diff
@@ -29,11 +59,7 @@ module.exports = function karma(config) {
     reporters: ['dots'],
 
     files: [
-      'test/*.{jsx,js}',
-    ],
-
-    exclude: [
-      'test/_*.{jsx,js}',
+      'test/*-spec.{jsx,js}',
```

Comment:
everything in the test folder, whether it has a "-spec" suffix or not, should be included, except the helpers that start with _ - let's revert this part for now.